fix(jans-tarp): fix tarp's build error

error.log

```

> janssen-tarp@1.15.0 build
> webpack --mode=production --progress --config webpack.prod.js

assets by status 5.22 MiB [cached] 23 assets
Entrypoint popup = 28.js popup.js
Entrypoint options = 28.js 591.js options.js 1 auxiliary asset
Entrypoint background = background.js
orphan modules 3.04 MiB [orphan] 1252 modules
runtime modules 12.4 KiB 19 modules
cacheable modules 4.13 MiB (javascript) 3.62 MiB (asset)
  modules by path ./node_modules/ 3.93 MiB (javascript) 3.62 MiB (asset) 389 modules
  modules by path ./src/ 209 KiB
    modules by path ./src/static/css/*.css 6.42 KiB
      ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[1].use[1]!./node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[1].use[2]!./src/static/css/popup.css 588 bytes [built] [code generated]
      + 2 modules
    modules by path ./src/ai/ 6.11 KiB
      ./src/ai/llm/OpenAIClient.ts 802 bytes [built] [code generated]
      + 2 modules
    ./src/popup/index.tsx + 2 modules 2.07 KiB [built] [code generated]
    ./src/options/index.tsx + 34 modules 194 KiB [built] [code generated]
    ./src/background/background.ts 170 bytes [built] [code generated]
  ./util.inspect (ignored) 15 bytes [built] [code generated]

ERROR in ./node_modules/@mui/x-date-pickers/AdapterMoment/AdapterMoment.js 3:0-35
Module not found: Error: Can't resolve 'moment' in '/home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/@mui/x-date-pickers/AdapterMoment'
resolve 'moment' in '/home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/@mui/x-date-pickers/AdapterMoment'
  Parsed request is a module
  using description file: /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/@mui/x-date-pickers/AdapterMoment/package.json (relative path: .)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/@mui/x-date-pickers/AdapterMoment/node_modules doesn't exist or is not a directory
      /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/@mui/x-date-pickers/node_modules doesn't exist or is not a directory
      /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/@mui/node_modules doesn't exist or is not a directory
      /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/node_modules doesn't exist or is not a directory
      looking for modules in /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules
        single file module
          using description file: /home/work/work/jans/demos/janssen-tarp/browser-extension/package.json (relative path: ./node_modules/moment)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/moment doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/moment.tsx doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/moment.js doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/moment.ts doesn't exist
        /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/moment doesn't exist
      /home/work/work/jans/demos/janssen-tarp/node_modules doesn't exist or is not a directory
      /home/work/work/jans/demos/node_modules doesn't exist or is not a directory
      /home/work/work/jans/node_modules doesn't exist or is not a directory
      /home/work/work/node_modules doesn't exist or is not a directory
      /home/work/node_modules doesn't exist or is not a directory
      /home/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
 @ ./node_modules/@mui/x-date-pickers/AdapterMoment/index.js 1:0-48 1:0-48
 @ ./src/options/registerClient.tsx 12:0-66 329:77-90
 @ ./src/options/oidcClients.tsx 17:0-46 109:28-42
 @ ./src/options/homePage.tsx 9:0-40 51:32-43
 @ ./src/options/options.tsx 4:0-34 41:43-51 43:43-51
 @ ./src/options/index.tsx 3:0-32 13:36-43

webpack 5.104.1 compiled with 1 error in 14664 ms

assets by status 5.22 MiB [cached] 23 assets
Entrypoint popup = 28.js popup.js
Entrypoint options = 28.js 591.js options.js 1 auxiliary asset
Entrypoint background = background.js
orphan modules 3.04 MiB [orphan] 1252 modules
runtime modules 12.4 KiB 19 modules
cacheable modules 4.13 MiB (javascript) 3.62 MiB (asset)
  modules by path ./node_modules/ 3.93 MiB (javascript) 3.62 MiB (asset) 389 modules
  modules by path ./src/ 209 KiB
    modules by path ./src/static/css/*.css 6.42 KiB
      ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[1].use[1]!./node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[1].use[2]!./src/static/css/popup.css 588 bytes [built] [code generated]
      + 2 modules
    modules by path ./src/ai/ 6.11 KiB
      ./src/ai/llm/OpenAIClient.ts 802 bytes [built] [code generated]
      + 2 modules
    ./src/popup/index.tsx + 2 modules 2.07 KiB [built] [code generated]
    ./src/options/index.tsx + 34 modules 194 KiB [built] [code generated]
    ./src/background/background.ts 170 bytes [built] [code generated]
  ./util.inspect (ignored) 15 bytes [built] [code generated]

ERROR in ./node_modules/@mui/x-date-pickers/AdapterMoment/AdapterMoment.js 3:0-35
Module not found: Error: Can't resolve 'moment' in '/home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/@mui/x-date-pickers/AdapterMoment'
resolve 'moment' in '/home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/@mui/x-date-pickers/AdapterMoment'
  Parsed request is a module
  using description file: /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/@mui/x-date-pickers/AdapterMoment/package.json (relative path: .)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/@mui/x-date-pickers/AdapterMoment/node_modules doesn't exist or is not a directory
      /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/@mui/x-date-pickers/node_modules doesn't exist or is not a directory
      /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/@mui/node_modules doesn't exist or is not a directory
      /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/node_modules doesn't exist or is not a directory
      looking for modules in /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules
        single file module
          using description file: /home/work/work/jans/demos/janssen-tarp/browser-extension/package.json (relative path: ./node_modules/moment)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/moment doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/moment.tsx doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/moment.js doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/moment.ts doesn't exist
        /home/work/work/jans/demos/janssen-tarp/browser-extension/node_modules/moment doesn't exist
      /home/work/work/jans/demos/janssen-tarp/node_modules doesn't exist or is not a directory
      /home/work/work/jans/demos/node_modules doesn't exist or is not a directory
      /home/work/work/jans/node_modules doesn't exist or is not a directory
      /home/work/work/node_modules doesn't exist or is not a directory
      /home/work/node_modules doesn't exist or is not a directory
      /home/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
 @ ./node_modules/@mui/x-date-pickers/AdapterMoment/index.js 1:0-48 1:0-48
 @ ./src/options/registerClient.tsx 12:0-66 329:77-90
 @ ./src/options/oidcClients.tsx 17:0-46 109:28-42
 @ ./src/options/homePage.tsx 9:0-40 51:32-43
 @ ./src/options/options.tsx 4:0-34 41:43-51 43:43-51
 @ ./src/options/index.tsx 3:0-32 13:36-43

webpack 5.104.1 compiled with 1 error in 14664 ms
```

--- a/demos/janssen-tarp/browser-extension/package.json
+++ b/demos/janssen-tarp/browser-extension/package.json
@@ -45,6 +45,7 @@
     "dayjs": "^1.11.19",
     "json-edit-react": "^1.19.2",
     "jwt-decode": "^4.0.0",
+    "moment": "^2.30.1",
     "openai": "^6.9.1",
     "postcss": "^8.4.14",
     "qs": "^6.11.2",

